neuromation/api/parser.py

```diff
@@ -64,3 +108,61 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
             self._config.username, self._config.cluster_name, self._config.registry_url
         )
         return parser.convert_to_local_image(image)
+
+    def env(
+        self, env: Sequence[str], env_file: Sequence[str] = ()
+    ) -> Tuple[Dict[str, str], Mapping[str, URL]]:
+        env_dict = self._build_env(env, env_file)
+        secret_env_dict = self._extract_secret_env(env_dict)
+        return env_dict, secret_env_dict
+
+    def _build_env(
+        self, env: Sequence[str], env_file: Sequence[str] = ()
+    ) -> Dict[str, str]:
+        lines: List[str] = []
+        for filename in env_file:
+            lines.extend(_read_lines(filename))
+        lines.extend(env)
+
+        env_dict = {}
+        for line in lines:
+            splitted = line.split("=", 1)
+            name = splitted[0]
+            if len(splitted) == 1:
+                val = os.environ.get(splitted[0], "")
+            else:
+                val = splitted[1]
+            if name in RESERVED_ENV_VARS:
+                raise ValueError(
+                    f"Unable to re-define system-reserved environment variable: {name}"
+                )
```

Comment:
Why forbid this?

Reply:
This has been the case historically. I'm afraid I don't have context. Perhaps @yartem can help?

Reply:
It has been to prevent conflicts with `--volume=ALL` and `--volume=HOME`. But since these features are no longer supported these environment variables are no longer set by client.

Reply:
Ah, I see. Removed it, as well as corresponding docs and tests.